unit-test-js/karma.conf.js

```diff
@@ -0,0 +1,29 @@
+module.exports = function (config) {
+    config.set({
+
+        basePath: '../',
+
+        files: [
+            'static/libs/angular/js/angular.min.js',
+            'static/libs/angular-mocks/js/angular-mocks.js',
+            'static/js/rdm.js',
+            'unit-test-js/e2e-tests/scenarios.js'
```

Comment:
It's strange to have e2e tests inside unit-tests, since they are two different types of testing. 

I'd also break out one file per controller, so I think it would be best to change unit-test-js/e2e-tests/scenarios.js to unit-test-js/rdm_test.js